src/test/resources/org/folio/processing/mapping/mappedAuthorityRecord.json

```diff
@@ -0,0 +1,57 @@
+{
+  "id": "0",
```

Comment:
Why the id is 0. It should be an uuid from 999 field: b90cb1bc-601f-45d7-b99e-b11efd281dcd